src/components/EditDeck/EditDeck.js

```diff
@@ -412,9 +412,9 @@ class EditDeck extends Component {
             {this.state.cardSearchInput.length >= 1 ? (
               <div className="optionsDiv">
                 <ul className="cardOptions">
-                  {this.props.reduxStore.card.map((option) => (
+                  {this.props.reduxStore.card.map((option, i) => (
                     <li
-                      key={option.id}
+                      key={`${i + 1}${option.id}`}
```

Comment:
Best practice is to not include indexes in keys. Most linters will warn you about it. BUT in this case it was a quick an easy way to get the error to go away so the render would work